presto-orc/src/test/java/com/facebook/presto/orc/TestDwrfDictionaries.java

```diff
@@ -41,9 +42,18 @@
     public void testSlice()
             throws Exception
     {
-        Block block1 = readFile("test_dictionaries/string-dictionary.orc", VarcharType.VARCHAR);
-        Block block2 = readFile("test_dictionaries/string-row-group-dictionary.dwrf", VarcharType.VARCHAR);
-        compare(VarcharType.VARCHAR, block1, block2);
+        Block block1 = readFile("test_dictionaries/string-dictionary.orc", VARCHAR);
```

Comment:
- block1 and block2 are confusing as it is not clear which is being tested. I'd rename these to expected and actual.
- I'd also rename compare method to assertEqualBlock
- I'd also inline null1 and null2 variables

```
    @Test
    public void testSlice()
            throws Exception
    {
        Block expected = readFile("test_dictionaries/string-dictionary.orc", VARCHAR);
        Block actual = readFile("test_dictionaries/string-row-group-dictionary.dwrf", VARCHAR);
        assertEqualBlocks(VARCHAR, expected, actual);
    }

    private static void assertEqualBlocks(Type type, Block expected, Block actual)
    {
        assertEquals(expected.getPositionCount(), actual.getPositionCount());
        for (int i = 0; i < expected.getPositionCount(); i++) {
            assertEquals(actual.isNull(i), expected.isNull(i));
            if (!expected.isNull(i)) {
                assertEquals(type.compareTo(expected, i, actual, i), 0);
            }
        }
    }
```